### ISSUE TITLE
New release card in release notes announcing Beta test version for Employer Access

### ISSUE THREAD

**Issue by camerojo** [talentbeyondboundaries/tbbtalentv2#1382]
### Proposed text.

In this release, Employer Access to the Talent Catalog is finally ready for beta testing with a real employer.

Here is a [good definition of beta testing](https://www.productplan.com/glossary/beta-test/).

It has been a long journey to get to this point, largely because employers will not have access to products that we have traditionally used alongside the Talent Catalog - in particular our Salesforce and Slack. Using those products - which we have done from the very beginning of TBB - allowed us to quickly deliver useful functionality without having to write everything ourselves. TBB will continue to use Salesforce and Slack - but Employers (and other third parties) need to be able to use the TC without needing to share TBB's Salesforce and Slack. This has driven fundamental changes to the TC code including the creation of TC Chats (formerly known as Job Chats), and changing the user interface so that the TBB's Salesforce is hidden from the user, and all process updates - such as updating stages and next steps - happen on the TC.

Employer access is also a very major step towards Recruiter access where third party recruitment agencies, such as Jumpstart and Talent Lift in Canada can also have direct access to the TC. 

Initially we will be testing with just one real employer: Iress. There is no employer that we have a closer or longer relationship with than Iress. That is important in beta testing. We expect that the process will highlight bugs and shortcomings that have not come up in our own testing. It is important to beta test with users who understand that and will not be disappointed when problems occur. The fact that Iress develop software themselves is a big advantage. They understand software and the whole concept of beta testing.

Once the initial testing with Iress is complete - we will engage with other employers that we have a close relationship with to complete our beta testing.

**Comment by camerojo** [comment]
### TBB New Job screen

<img width="1235" alt="Screenshot 2024-09-18 at 10 14 12 AM" src="https://github.com/user-attachments/assets/69d56d65-0860-45bc-aa73-009a2d1964c8">

### Employer access New Job screen (Iress)

<img width="1236" alt="Screenshot 2024-09-18 at 10 13 51 AM" src="https://github.com/user-attachments/assets/13ba740b-25e2-4ff9-ac17-c3cff960ac92">

**Comment by camerojo** [comment]
@sadatmalik - can you review the above

**Comment by cazcam34** [comment]
Completed by @sadatmalik and as mentioned in Slack I updated the image to include Iress logo, I also updated the other TBB image to show TBB logo to keep consistent. 
In addition created new issue and PR to add Iress Logo to tc so we can update this when release is out (see #1389) have added this to release planning
